src/lib/routing-tables.js

```diff
@@ -56,7 +56,7 @@ class RoutingTables {
    * @param {Route|RouteData} _route from ledger B→C
    * @returns {Boolean} whether or not a new route was added
    */
-  addRoute (_route) {
+  addRoute (_route, prefix) {
```

Comment:
I like destinationPrefix because it's the key added to the `PrefixMap` named `destinations`. The route constructor already defaults destinationPrefix to destinationLedger, I believe: https://github.com/interledgerjs/ilp-routing/pull/13/files#diff-461b64a626e26d6495dc1eed9f8787d5R92